src/app/suggestion-notifications/reciter-suggestions/suggestion-actions/suggestion-actions.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { ItemType } from '../../../core/shared/item-relationships/item-type.model';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { Suggestion } from '../../../core/suggestion-notifications/reciter-suggestions/models/suggestion.model';
+import { SuggestionApproveAndImport } from '../suggestion-list-element/suggestion-list-element.component';
+import { Collection } from '../../../core/shared/collection.model';
+import { take } from 'rxjs/operators';
+import { CreateItemParentSelectorComponent } from '../../../shared/dso-selector/modal-wrappers/create-item-parent-selector/create-item-parent-selector.component';
+
+@Component({
+  selector: 'ds-suggestion-actions',
+  styleUrls: [ './suggestion-actions.component.scss' ],
+  templateUrl: './suggestion-actions.component.html'
+})
+export class SuggestionActionsComponent {
+
+  @Input() object: Suggestion;
+
+  @Input() isBulk = false;
+
+  @Input() hasEvidence = false;
+
+  @Input() seeEvidence = false;
+
+  @Input() isCollectionFixed = false;
+
+  /**
+   * The component is used to Delete suggestion
+   */
+  @Output() notMineClicked = new EventEmitter<string>();
```

Comment:
I'd recommend renaming all the "notMine" code.  It's not the most descriptive, as Admins can be accepting/rejecting these suggestions _on behalf of_ other Researchers.  I think we should use a phrase like "rejectSuggestion" or "ignoreSuggestion" or "deleteSuggestion" or something similar to that.  All of those will make more sense than "not mine"